sqlagg/base.py

```diff
@@ -175,6 +179,12 @@ def table_filter(table_name, metadata):
 
         return self._metadata
 
+    def count(self, connection, filter_values=None):
```

Comment:
I'm actually intending this to replace https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/reports/data_source.py#L158 which I'm prioritizing since it's called for all UCRs (the total row is only available internally at the moment)